Fika.Core/Coop/Components/CoopHandler.cs

```diff
@@ -348,7 +347,7 @@ private async void SpawnPlayer(SpawnObject spawnObject)
                     if (botController != null)
                     {
                         // Start Coroutine as botController might need a while to start sometimes...
-                        Logger.LogInfo("Starting AddClientToBotEnemies routine.");
+                        // Logger.LogInfo("Starting AddClientToBotEnemies routine.");
```

Comment:
```suggestion
#if DEBUG
                        Logger.LogInfo("Starting AddClientToBotEnemies routine.");
#endif
```
Rather than removing all these, can you include them in the` DEBUG` build?